src/main/java/org/apereo/portlet/soffit/model/v1_0/Group.java

```diff
@@ -19,7 +19,12 @@
 
 package org.apereo.portlet.soffit.model.v1_0;
 
-public class Role {
+/**
+ * Represents a class of users in the portal.
+ *
+ * @author drewwills
+ */
+public class Group {
```

Comment:
Changed name from `Role` to `Group` in an attempt to signify that these things aren't intended (precisely) for authorization purposes.  They are portal constructs -- some groups merely indicate the class of device you're using or the theme in which you're viewing the portal.  We have another plan (coming) for AuthN.